src/identity/components/GlobalHeader/IdentityUserAvatar.tsx

```diff
@@ -88,7 +88,7 @@ class IdentityUserAvatar extends React.Component<Props, State> {
           <Link
             className="user-popover-footer--button"
             onClick={this.handlePopoverClick(HeaderNavEvent.UserLogoutClick)}
-            to="/logout"
+            to="/logoutyeah"
```

Comment:
Is this the correct path?

Reply:
My bad, all testing changes snuck in as well, fixed now!